src/dune_rules/simple_rules.mli

```diff
@@ -13,6 +13,19 @@ module Alias_rules : sig
 
   val add_empty :
     Super_context.t -> loc:Stdune.Loc.t option -> alias:Alias.t -> unit Memo.t
+
+  val add_list :
+       Super_context.t
+    -> aliases:Alias.t list
+    -> loc:Loc.t option
+    -> Action.Full.t Action_builder.t
+    -> unit Memo.t
+
+  val add_list_empty :
+       Super_context.t
+    -> loc:Stdune.Loc.t option
+    -> aliases:Alias.t list
+    -> unit Memo.t
```

Comment:
Can we inline both of these functions? They're only used once so they're really just API bloat.

Reply:
Ah yes, I originally added them thinking I would need it elsewhere, but you are correct they are completely useless. I've removed the changes in the mli and inlined them now.